resources/tenant-onboarding.yaml

```diff
@@ -692,8 +815,24 @@ Resources:
                   Value: !GetAtt rds.Outputs.RdsEndpoint
                 - Name: DB_NAME
                   Value: !Ref RDSDatabase
+                - Name: ES_CLUSTER_ENDPOINT
```

Comment:
is it also not possible to add these new env vars directly under `Environment` as opposed to in both the if and else sections of the if statement on the ProvisionRDS condition?

also wouldn't it be more accurate to add these under their own "if ProvisionES" block and then join them with the others? or is that just really clunky to define in CFT